include/fi.h

```diff
@@ -209,8 +235,12 @@ int fi_rma_target_allowed(uint64_t caps);
         asm(".symver " #name "," #api "@@" DEFAULT_ABI)
 #else
 #  define symver(name, api, ver)
+#ifdef  HAVE_ALIAS_ATTRIBUTE
```

Comment:
I'll look into the simplification. The OS X release isn't expected to be a production release and will not support versioned symbols. Otherwise, it supports compilers without alias support.